microcloud/service/lxd_config.go

```diff
@@ -184,3 +184,45 @@ func (s LXDService) DefaultCephStoragePoolJoinConfig() api.ClusterMemberConfigKe
 		Value:  "lxd_remote",
 	}
 }
+
+// DefaultPendingCephFSStoragePool returns the default cephfs storage configuration when
+// creating a pending pool on a specific cluster member target.
+func (s LXDService) DefaultPendingCephFSStoragePool() api.StoragePoolsPost {
+	return api.StoragePoolsPost{
+		Name:   "remote-fs",
+		Driver: "cephfs",
+		StoragePoolPut: api.StoragePoolPut{
+			Config: map[string]string{
+				"source": "lxd_cephfs",
+			},
+		},
+	}
+}
+
+// DefaultCephFSStoragePool returns the default cephfs storage configuration when
+// creating the finalized pool.
+func (s LXDService) DefaultCephFSStoragePool() api.StoragePoolsPost {
+	return api.StoragePoolsPost{
+		Name:   "remote-fs",
+		Driver: "cephfs",
+		StoragePoolPut: api.StoragePoolPut{
+			Config: map[string]string{
+				"cephfs.create_missing": "true",
+				"cephfs.meta_pool":      "lxd_cephfs_meta",
+				"cephfs.data_pool":      "lxd_cephfs_data",
+			},
+			Description: "Remote CephFS file-system",
```

Comment:
To be in line with the description in `DefaultCephStoragePool()` maybe this can be changed to something like "Distributed file-system storage using CephFS"?